palera1n.sh

```diff
@@ -110,6 +110,10 @@ _pwn() {
 }
 
 _dfuhelper() {
+    echo "[*] To get into DFU mode, you will be guided through 2 steps:"
+    echo "[*] 1. Hold volume down + side button for 4 seconds"
+    echo "[*] 2. Release side button, then keep holding volume down for 10 seconds"
```

Comment:
I changed it from "but" to "then", because you're not releasing the side button for 10 seconds.

Reply:
Which only applies here, where I added the time in seconds.